src/main/java/ng/abdlquadri/eventbus/EventBusFrameHandler.java

```diff
@@ -46,7 +47,8 @@ public void run() {
 
   @Override
   public void channelInactive(ChannelHandlerContext ctx) throws Exception {
-    log.log(Level.SEVERE, "CHANNEL NOT ACTIVE");
+	// log OR throw exception, never both
```

Comment:
I think - after you made these alterations final- I'll add an onError() too and see how it works out, and I'll do a new pull request for that.